hdk-rust/tests/integration_test.rs

```diff
@@ -226,10 +227,18 @@ fn has_populated_validation_data() {
 fn can_link_entries() {
     let (mut hc, _) = start_holochain_instance();
 
-    //
-    // Add two entries to chain to have something to check ValidationData on
-    //
     let result = hc.call("test_zome", "test_cap", "link_two_entries", r#"{}"#);
     assert!(result.is_ok(), "\t result = {:?}", result);
     assert_eq!(result.unwrap(), r#"{"ok":true}"#);
 }
+
+#[test]
+fn can_roundtrip_links() {
+    let (mut hc, _) = start_holochain_instance();
+    let result = hc.call("test_zome", "test_cap", "links_roundtrip", r#"{}"#);
+    assert!(result.is_ok(), "\t result = {:?}", result);
```

Comment:
that `\t` doesn't seem necessary as the error message is wrapped in a panic format.